eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/impl/MultiPageEditor.java

```diff
@@ -836,19 +859,15 @@ protected SelectionChangedEvent getNewEvent(ISelectionProvider parentProvider, S
 
             protected StructuredSelection getNewSelection(Node node) {
                 int type = node.getNodeType();
-                if (type == 1) {
-                    // node type = element
-                    // Element elem = (Element) node;
+                if (type == 1 || type == 3) {
```

Comment:
Are there no constants defined in Node for those values? It would be more readable.